src/pages/injector/app.js

```diff
@@ -5,10 +5,32 @@ import { ItemPreviewConnector } from 'connectors/item-preview/item-preview'
 import { TaggingConnector } from 'connectors/tagging/tagging'
 import { FooterConnector } from 'connectors/footer/footer'
 
+import { SAVE_TO_POCKET_REQUEST } from 'actions'
+
 export const App = () => {
   const appTarget = useRef(null)
   const [isOpen, setIsOpen] = useState(false)
 
+  /* Handle incoming messages
+  –––––––––––––––––––––––––––––––––––––––––––––––––– */
+  const handleMessages = (event) => {
+    const { payload, action = 'Unknown Action' } = event || {}
+
+    console.groupCollapsed(`RECEIVE: ${action}`)
+    console.log(payload)
+    console.groupEnd(`RECEIVE: ${action}`)
+
+    switch (action) {
+      case SAVE_TO_POCKET_REQUEST: {
+        return setIsOpen(true)
+      }
+
+      default: {
+        return
+      }
+    }
+  }
+
```

Comment:
We needed to add this back for instances when a user clicks Save twice on the page